app/views/add-reduction.html

```diff
@@ -6,7 +6,11 @@
   {% include "includes/breadcrumbs.html" %} 
   
   <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">
-    New reduction
+    {% if reduction.reasonId !== undefined %}
```

Comment:
Why do you rely on the reason.id as opposed to the reduction id?

Reply:
No reason, changed to just look at the reduction, the entire thing is undefined if there is no existing reduction

Reply:
Moved all variables that are different between add and edit reduction to a single if at the top of the file